image_classifier_3d/data_loader/universal_loader.py

```diff
@@ -66,15 +61,15 @@ class adaptive_padding_loader(Dataset):
         In general, adaptive padding data loader will pad all images to the
         same size defined by "out_shape" when constructing the data loader.
         For training, random flip and rotaion will be applied. No augmentation
-        for testing or validation.
+        for testing or evaluation.
 
-        "test_flag" is a key parameter for determining how data loading
+        "flag" is a key parameter for determining how data loading
         works in different scenarios:
             * "T": trianing
-            * "F": validation on a folder of files
+            * "F": evaluation on a folder of files
             * "C": test on files listed in a CSV
 
-        For test_flag == "T" or "F" :
+        For flag == "T" or "F" :
 
             Assumption: all data should be saved in a folder with
             filenames 0_xxxxx.npy, 0_xxxxx.npy, 1_xxxx.npy, 1_xxxx.npy,
```

Comment:
Sorry, I didn't get this comment. Do 0_xxxxx.npy and 0_xxxxx.npy refer to the same image? Or you trying to say it can have any name as long as it starts with the class number?

Reply:
Good point!! I will try to reword this part.